.github/actions/emit-metrics/action.yml

```diff
@@ -0,0 +1,74 @@
+name: Emit Metrics
+description: Emit metrics from another workflow/action using the kat tool
```

Comment:
nit: "from another workflow/action" is implied, could be "Emit metrics using that kat tool"